nesta/production/schemas/tier_1/aliases/health_scanner.json

```diff
@@ -78,7 +86,8 @@
     },
     "state": {
         "nih": "placeName_state_organisation",
-        "crunchbase": "placeName_state_organisation"
+        "crunchbase": "placeName_state_organisation",
+        "meetup": "placeName_state_group"
     },
     "state_id": {
         "nih": "id_state_organisation",
```

Comment:
meetup doesn't have a `state` field unfortunately. This could be added in in the future, but will require some new dev for that. I've realised that `meetup_es_config.json` indicates that there is, so I've prefixed these with an underscore now.